dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Machines/Governors/Standard/Generic/GovCT2.mo

```diff
@@ -0,0 +1,357 @@
+  parameter Types.PerUnit RDownPu = -99 "Maximum rate of load limit decrease in pu/s (base PNomTurb*KTurb), typical: -99" annotation( Dialog(tab = "Load limit controller"));
+  parameter Types.PerUnit ROpenPu = 99 "Maximum valve opening rate in pu/s (base PNomTurb*KTurb), typical: 99" annotation( Dialog(tab = "Turbine/engine"));
+  parameter Types.PerUnit RDroop = 0.05 "Permanent droop, typical: 0.05" annotation( Dialog(tab = "Main control path"));
+  parameter Integer RSelectInt "Feedback signal for droop" annotation( Dialog(tab = "Main control path"));
+  parameter Types.PerUnit RUpPu = 99 "Maximum rate of load limit increase in pu/s (base PNomTurb*KTurb), typical: 99" annotation( Dialog(tab = "Load limit controller"));
+  parameter Types.Time tActuator = 0.4 "Actuator time constant in s, typical: 0.4" annotation( Dialog(tab = "Turbine/engine"));
+  parameter Types.Time tA = 1 "Acceleration limiter time constant in s, typical: 1" annotation( Dialog(tab = "Acceleration limiter"));
+  parameter Types.Time tB = 0.1 "Turbine lag time constant in s, typical: 0.1" annotation( Dialog(tab = "Turbine/engine"));
+  parameter Types.Time tC = 0 "Turbine lead time constant in s, typical: 0" annotation( Dialog(tab = "Turbine/engine"));
+  parameter Types.Time tDGov = 1 "Governor derivative controller time constant in s, typical: 1" annotation( Dialog(tab = "Main control path"));
+  parameter Types.Time tDRatelim = 0.001 "Ramp rate limter derivative time constant in s, typical: 0.001";
+  parameter Types.Time tEngine = 0 "Transport time delay for diesel engine in s, typical: 0" annotation( Dialog(tab = "Turbine/engine"));
+  parameter Types.Time tFLoad = 3 "Load limiter time constant in s, typical: 3" annotation( Dialog(tab = "Load limit controller"));
+  parameter Types.Time tLastValue = 1e-9 "Fast delay block time constant in s, typical: 1e-9" annotation( Dialog(tab = "Main control path"));
+  parameter Types.Time tPElec = 2.5 "Electrical power transducer time constant in s, typical: 2.5" annotation( Dialog(tab = "Main control path"));
+  parameter Types.Time tSA = 1e-6 "Temperature detection lead time constant in s, typical: 1e-6" annotation( Dialog(tab = "Load limit controller"));
+  parameter Types.Time tSB = 50 "Temperature detection lag time constant in s, typical: 50" annotation( Dialog(tab = "Load limit controller"));
+  parameter Types.PerUnit ValveMaxPu = 1 "Maximum valve position limit in pu (base PNomTurb*KTurb), typical: 1" annotation( Dialog(tab = "Turbine/engine"));
+  parameter Types.PerUnit ValveMinPu = 0.175 "Minimum valve position limit in pu (base PNomTurb*KTurb), typical: 0.175" annotation( Dialog(tab = "Turbine/engine"));
+  parameter Types.PerUnit WFnlPu = 0.187 "No load fuel flow in pu (base PNomTurb*KTurb), typical: 0.187" annotation( Dialog(tab = "Turbine/engine"));
+  parameter Boolean WFSpdBool = false "Switch for fuel source characteristic, typical: false" annotation( Dialog(tab = "Turbine/engine")); 
```

Comment:
Yes it's disturbing in dynawo, we have seen problems when imposing a default value in the Modelica model, the change in the .par won't be taken into account.
That's why the parameters' value should be given in the .par only. 
since we put a typical value in the definition of each parameter, this can be useful to the user that is using it in OpenModelica.

Reply:
(I did add them now. please verify.)

Reply:
okay, I will remove the default values